tjreports/site/assets/js/tjrContentUI.js

```diff
@@ -45,6 +46,14 @@ jQuery.extend(tjrContentUI.report, {
 				var responseHTML = jQuery(response['html']).find(containerSel).html();
 				jQuery(containerSel).html(responseHTML);
 
+				// If sendEmail plug is enable then call the below funcation. To show checkboxes
+				var isSendEmail = jQuery('body').find('.td-sendemail').length;
+				if (isSendEmail)
```

Comment:
Also add a check for tjutilitysendemail defined to handle cases when the plugin is unpublished or not present.